src/App.scss

```diff
@@ -14,5 +15,23 @@ html, body, textarea, button {
 }
 
 html {
-  @include text-regular;
+  // @include text-regular;
+  box-sizing: border-box;
+  min-height: 100vh;
```

Comment:
Should we change this to be absolute units to avoid potentially having a min-height that is too small (e.g. it is possible to have min-height 0 with this).

Reply:
As discussed, this will set the overall viewport but not affect fixed sizing of child elements. i.e. if you have an element with 400px height, it will still be 400px. It will just allow relative sizing.